docs/Measuring_Resonances.md

```diff
@@ -450,7 +450,7 @@ TEST_RESONANCES AXIS=Y
 ```
 This will generate 2 CSV files (`/tmp/resonances_x_*.csv` and
 `/tmp/resonances_y_*.csv`). These files can be processed with the stand-alone
-script on a Raspberry Pi. To do that, run the following commands:
+script on a Raspberry Pi. To do that, run the following commands (WARNING: Do not run these commands when there is more than one .csv file in your tmp folder. This will result in ALL files being used for the charted results):
```

Comment:
> Do not change this line: grammatically brackets are NOT meant to include whole sentences. If it's more than few words, it's not for brackets.
> 
> Add a line before line 458, after the code, as suggested:
> 
> ```
> 457bis Keep in mind that these commands (written with the "*") will merge all available CSV files
> from the specified folder to generate a single calibration graph. It may be useful if you tested the
> machine in various conditions and you want a single result which takes all of them into account.
> ```
> 
> Remember to split at column 80, as the file originally was, don't add long lines which make the code difficult to read. Lines are free of charge :)

I don't fully agree with this edit. There's no part of the documentation that already tells or assumes the user should or could use multiple CSV files, so the inherent assumption here is that the user is creating a single CSV. Given that, I don't think we should say things like "keep in mind", but more of a warning like "do not run this with multiple CSVs in your folder, unless you intend to blend the results".

I assume the original intent of using a wildcard was to save the user time from writing out the file name, but again it's not clear from the documentation.

Will submit a modified commit.